main.cpp

```diff
@@ -79,42 +101,100 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+bool biggerThan(int x, int y)
+{
+    ignoreUnused(x, y);
+    return {};
+}
+
 
 /*
  2)
- */
+ */
+int numOfLegs(char* animal, int numOfArms = 2)
```

Comment:
Pointers are covered in Project 6. 
Do not use pointers until they are covered as you do not know the risks that come with using them.
- [ ] revise